docs/source/examples/cirq-ibmq-backends.myst

```diff
@@ -0,0 +1,264 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+(label-cirq-zne-example)=
+# Error mitigation on IBMQ backends with Cirq
+
+
+This tutorial shows an example of how to mitigate noise on IBMQ
+backends with the Cirq frontend, broken down in the following steps.
+
+- [Error mitigation on IBMQ backends with Cirq](#error-mitigation-cirq-on-ibmq-backends)
+  - [Settings](#settings)
+  - [Setup: Defining a circuit](#setup-defining-a-circuit)
+  - [High-level usage](#high-level-usage)
+  - [Options](#options)
+  - [Lower-level usage](#lower-level-usage)
+
++++
+
+## Settings
+
+```{code-cell} ipython3
+from mitiq import zne
+from mitiq.interface.mitiq_qiskit.qiskit_utils import initialized_depolarizing_noise
+```
+
+**Note:** If `USE_REAL_HARDWARE` is set to `False`, a classically simulated noisy backend is used instead of a real quantum computer.
+
+```{code-cell} ipython3
+USE_REAL_HARDWARE = False
+```
```

Comment:
I think it makes sense to swap the order of these two so the note comes after the definition. WDYT?

Reply:
If we do make this change, the note should read `**Note:** When...`

Reply:
Agree, it flows better with the note afterwards.